pkg/pillar/types/global.go

```diff
@@ -890,12 +925,22 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	return configItemSpecMap
 }
 
-// parseLevel - Wrapper that ignores the 'Level' output of the logrus.ParseLevel function
-func parseLevel(level string) error {
+// parseLogrusLevel - Wrapper for parsing logrus loglevel
+func parseLogrusLevel(level string) error {
```

Comment:
Should `parse*Level` functions be actually called `validate*Level`? They do not return the parsed value, just the error.

Reply:
Yeah, why not. Can change that.

Reply:
So let's do it :)

Reply:
Waiting for the correct moment!!11